core/opendaq/signal/include/opendaq/signal_impl.h

```diff
@@ -688,23 +688,30 @@ ConstCharPtr SignalBase<TInterface, Interfaces...>::SerializeId()
 }
 
 template <typename TInterface, typename... Interfaces>
-ErrCode SignalBase<TInterface, Interfaces...>::Deserialize(ISerializedObject* serialized, IBaseObject* context, IFunction* factoryCallback, IBaseObject** obj)
+ErrCode SignalBase<TInterface, Interfaces...>::Deserialize(ISerializedObject* serialized,
+                                                           IBaseObject* context,
+                                                           IFunction* factoryCallback,
+                                                           IBaseObject** obj)
 {
     OPENDAQ_PARAM_NOT_NULL(obj);
     return daqTry(
         [&obj, &serialized, &context, &factoryCallback]()
         {
-            *obj = Super::DeserializeComponent(
-                       serialized,
-                       context,
-                       factoryCallback,
-                       [](const SerializedObjectPtr& serialized,
-                          const ComponentDeserializeContextPtr& deserializeContext,
-                          const StringPtr& className)
-                       {
-                           return createWithImplementation<ISignalConfig, SignalImpl>(
-                               deserializeContext.getContext(), nullptr, deserializeContext.getParent(), deserializeContext.getLocalId(), className);
-                       }).detach();
+            *obj =
+                Super::DeserializeComponent(serialized,
+                                            context,
+                                            factoryCallback,
+                                            [](const SerializedObjectPtr& serialized,
+                                               const ComponentDeserializeContextPtr& deserializeContext,
+                                               const StringPtr& className)
+                                            {
+                                                return createWithImplementation<ISignalConfig, SignalImpl>(deserializeContext.getContext(),
+                                                                                                           nullptr,
+                                                                                                           deserializeContext.getParent(),
+                                                                                                           deserializeContext.getLocalId(),
+                                                                                                           className);
+                                            })
+                    .detach();
```

Comment:
I would sometimes maybe check if our clang format is really the best it could be. I personally don't think it handles lambdas all that well - I preferred the previous version. 

We should probably take a look at our definition.